_includes/hemline.html

```diff
@@ -25,6 +25,7 @@
 <link rel="stylesheet" href="{{ '/assets/css/academicons.min.css' | prepend: site.baseurl | prepend: site.url }}">
 
 <!-- Google Analytics -->
```

Comment:
Please move the `<!-- Google Analytics -->` comment inside the if statement.

Reply:
That's done now